ups/eupspkg.cfg.sh

```diff
@@ -17,20 +17,20 @@ install()
     case $(uname -s) in
         Linux*)
             ana_platform="Linux-x86_64"
-            conda_packages="conda_packages-linux-64.txt"
+            conda_packages="conda3_packages-linux-64.txt"
```

Comment:
I don't know where this text file comes from, but that could be a breaking change.